src/sfdxProject.ts

```diff
@@ -57,6 +96,10 @@ export class SfdxProjectJson extends ConfigFile<ConfigFile.Options> {
     if (upperCaseKey) {
       throw SfdxError.create('@salesforce/core', 'core', 'InvalidJsonCasing', [upperCaseKey, this.getPath()]);
     }
+
+    if (env.getBoolean('SFDX_SCHEMA_VALIDATE', false)) {
```

Comment:
We could also call `this.schemaValidate` no matter what, but only throw if this is set to true, otherwise, display a warning?

Reply:
Yeah definitely.  I'll make that change.